### PR TITLE
MA1-418: [Android 10/11]Live streaming volume is very low as it is heard only in the ear piece speaker

### PR DESCRIPTION
**Pull request by bvic23**
**Steps to reproduce:**

1. Log in to the app 
2. Tap on any camera
3. Start live streaming

**Expected:** 

Live streaming should be played through the media speaker. 

**Actual:**

The volume is very low as it is heard only in the ear piece speaker and can be heard only if the phone is kept very close to the ears

**Note:**

This is now observed in all the phones running on Android 10 and 11. 

https://jira.bgchtest.info/browse/MA1-418

### CODE REVIEW

File: android/src/main/java/com/oney/WebRTCModule/WebRTCModule.java [hunk 1]
```diff
@@ -75,6 +76,10 @@ public WebRTCModule(ReactApplicationContext reactContext, Options options) {
         ThreadUtils.runOnExecutor(() -> initAsync(options));
     }
 
+    public Activity getCurrentActivityHack() {
+        return super.getCurrentActivity();
```
Comment (bvic23) [inline android/src/main/java/com/oney/WebRTCModule/WebRTCModule.java:80]: `getCurrentActivity` is protected in `super` and I need it in order to acquire AudioManager above.

**Comment by sesm** [comment]
Let's go with this a temporary solution. We already had a similar solution (using `react-native-loudspeaker`), and we had to revert it, see https://jira.bgchtest.info/browse/HCM-883

**Comment by sesm** [comment]
There are 2 problems with this solution: 
- audio will play through loud speaker even when headphones are connected (see https://jira.bgchtest.info/browse/HCM-883)
- call volume slider will be used (instead of media slider) for controlling the volume.

Current fix (adding `USAGE_MEDIA` and `FLAG_AUDIBILITY_ENFORCED` to audio track) addresses both issues, but doesn't work on some AndroidX devices.

So if we push this fix to all devices, this will be a regression for pre-AndroidX users.

Let's limit the scope of this fix to AndroidX, and push this as a temporary fix for AndroidX only.

**Review by sesm — APPROVED** [review]

**Comment by sesm** [comment]
👍

**Pull request merged**